kubernetes/samples/scripts/create-weblogic-domain/model-in-image/k8s-domain.yaml.template

```diff
@@ -90,42 +83,31 @@ spec:
     replicas: 2
   # The number of managed servers to start for unlisted clusters
   # replicas: 1
-  
-  # Optional configmap for additional models and variable files
-  wdtConfigMap : domain1-wdt-config-map
-
-  # user model-in-image online update or not (default is false)
-  #useOnlineUpdate: "true"
-
-  #
-  # For life cycle online update, if it is set to true it will cancel the update and error out
-  # default is false if not set
-  #
-  #rollbackIfRequireStart: true
 
-  #
-  # The wdt domain type for model in image case.
-  # Valid values are 'WLS', 'JRF', and 'RestrictedJRF'
-  #
-  wdtDomainType: "@@DOMTYPE@@"
+  configuration:
+    secrets: [ sample-domain1-rcu-access ]
 
-  #
-  #
-  # Keep jrf schema between updates. If not set, defaults to true.
-  #
-  #keepJRFSchema: "true"
+    model:
+      #
+      # Optional configmap for additional models and variable files
+      #
+      configMapName: wdt-config-map
 
-  #
-  # k8 secret name for the opss extract wallet.  It contains the key passphrase used to extract the
-  # opss wallet and optional key ewallet.p12 containing the base64 encoded opss wallet
-  #
-  #
-  #opssWalletSecret:
-  #  name: sample-domain1-opss-key-passphrase-secret
+      #
+      # wdt domain type for model in image case
+      # Valid WDT_DOMAIN_TYPE_VALUES are 'WLS', 'JRF', and 'RestrictedJRF'
+      #
+      domainType: "@@DOMTYPE@@"
 
-  #
-  # Use this if the wdt model is encrypted
-  #
-  #wdtEncryptionPassPhrase:
-  #  name: wdt-encrypt-passphrase-secret
+    #opss:
+      #
+      # k8 secret name for the opss extract wallet.  It contains the key passphrase used to extract the
+      # opss wallet and optional key ewallet.p12 containing the base64 encoded opss wallet
```

Comment:
Please update this description and related descriptions elsewhere to make it clear that we currently only use the opss settings if domain source type is 'FromModel'.   We'll need to add support for Image and PV cases later (and probably very soon since support for re-using key/wallet is critical even in the non-Model case), but I assume that's not a blocking goal for the model-in-image project.  The Image and PV cases should have a tracking JIRA.

Reply:
Let's handle this later as this template is only for model in image.